Ontology/Willow/Capability/Sensor/QuantitySensor/EnergySensor/Thermal/Billed/BilledThermalEnergyDemand.json

```diff
@@ -0,0 +1,17 @@
+{
+  "@id": "dtmi:com:willowinc:BilledThermalEnergyDemand;1",
```

Comment:
May not need all of these BilledThermalEnergyDemand models if the invidividual substances (i.e. Chilled Water, Steam) don't extend them.